ballsdex/settings.py

```diff
@@ -167,6 +168,7 @@ def read_settings(path: "Path"):
     settings.admin_guild_ids = content["admin-command"]["guild-ids"] or []
     settings.root_role_ids = content["admin-command"]["root-role-ids"] or []
     settings.admin_role_ids = content["admin-command"]["admin-role-ids"] or []
+    settings.admin_channel_ids = content["admin-command"].get("channel-ids", []) or []
```

Comment:
```suggestion
    settings.admin_channel_ids = content["admin-command"].get("admin-channel-ids", []) or []
```

The var, json-schema, and template config file have it as admin-channel-ids, so I presume you meant this (otherwise it doesn't load the channel ids)